devops/scripts/benchmarks/git_project.py

```diff
@@ -20,14 +20,16 @@ def __init__(
         name: str,
         force_rebuild: bool = False,
         no_suffix_src: bool = False,
+        shallow_clone: bool = False,
```

Comment:
why can't this be the default? I think we only need the exact commit/tag?

Reply:
Done